src/api/recordlinker/routes/demo_router.py

```diff
@@ -0,0 +1,55 @@
+"""
+recordlinker.routes.demo_router
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the demo router for the RecordLinker API. Exposing
+the API endpoints for the demo UI.
+"""
+
+import typing
+
+import fastapi
+
+from recordlinker import schemas
+from recordlinker.utils import path as utils
+
+router = fastapi.APIRouter()
+
+# Load static data for the Match Queue page in the demo UI
+data = utils.read_json("assets/demo_data.json")["demo_data"]
+
+
+def filter_and_sort(
+    data: typing.Dict,
+    status: typing.Optional[schemas.demo.LinkedStatus] = None,
+) -> typing.List[schemas.demo.MatchQueueRecord]:
+    """
+    Filter data by linked status and sort by received_on date.
+    """
+    status_filters = {
+        schemas.demo.LinkedStatus.linked: lambda d: d["linked"] is True,
+        schemas.demo.LinkedStatus.unlinked: lambda d: d["linked"] is False,
+        schemas.demo.LinkedStatus.evaluated: lambda d: d["linked"] is not None,
+        schemas.demo.LinkedStatus.pending: lambda d: d["linked"] is None,
+    }
+
+    filtered = [d for d in data if status_filters[status](d)] if status else data
+    return sorted(
+        filtered,
+        key=lambda x: x["received_on"],
```

Comment:
I had written "descending order" in #310, however I just took a look at the figma designs and the table shows ascending order.  @johanna-skylight can you confirm that the team is looking for the records to be sorted such that the oldest is at the top of the table?